### PR TITLE
Update to v4.1.0 alpha.4

### PR DESCRIPTION
**Pull request by justinmanley**
Pull latest changes from loaders.gl in order to include [recent implicit tiles fix](https://github.com/visgl/loaders.gl/pull/2844).

### CODE REVIEW

File: modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.ts [hunk 1]
```diff
@@ -109,7 +113,10 @@ export function normalizeTileData(
   let tileContentUrl: string | undefined;
   if (tile.content) {
     const contentUri = tile.content.uri || tile.content?.url;
-    tileContentUrl = resolveUri(contentUri, basePath);
+    if (typeof contentUri !== 'undefined') {
```
Comment (Kaapp) [inline modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.ts:110]: This hasn't quite automerged the change I made on Ib's request, there should be another removal in this file - would you rather I PR separately after this goes in or push a tiny commit on the end of this?

**Review by Rennzie — APPROVED** [review]
Approving assuming CKs comment is addressed.

File: modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.ts (hunk 1 above)
Comment (justinmanley) [reply modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.ts:110]: oops, sorry! that's my mistake -- I mixed up the two sides of the merge. Should be fixed now! Thanks for catching this! 🙇

**Review by Kaapp — APPROVED** [review]

**Comment by quipo** [comment]
superseded by https://github.com/sensat/loaders.gl/pull/9

**Pull request closed without merging**